tests/mysql.ini.tmpl

```diff
@@ -12,10 +12,11 @@ SSL_MODE = disable
 [indexer]
 REPO_INDEXER_ENABLED = true
 REPO_INDEXER_PATH = tests/{{TEST_TYPE}}/gitea-{{TEST_TYPE}}-mysql/indexers/repos.bleve
+ISSUE_INDEXER_TYPE = elasticsearch
+ISSUE_INDEXER_CONN_STR = http://elastic:changeme@elasticsearch:9200
 
 [queue.issue_indexer]
-TYPE = elasticsearch
-CONN_STR = http://elastic:changeme@elasticsearch:9200
+TYPE = level
```

Comment:
So the `TYPE` here will override setting on `[indexer]`?

Reply:
Yes, that's a longstanding bug ....

The "elasticsearch" test seems to have been broken for long time.